src/rebar_file_utils.erl

```diff
@@ -516,6 +517,24 @@ split_dirname(Path) ->
 ensure_dir(Path) ->
     filelib:ensure_dir(filename:join(Path, "fake_file")).
 
+%% returns the path for printing to logs. if get_cwd succeeds it
+%% attempts to return the relative path. It falls back on juts
+%% returning the original path.
+print_path(OriginalPath) ->
+   case file:get_cwd() of
+       {ok, Cwd} ->
+           case path_from_ancestor(OriginalPath, Cwd) of
+               {ok, ""} ->
+                   OriginalPath;
+               {ok, Path} ->
+                   Path;
+               _ ->
+                   OriginalPath
+           end;
+       _ ->
+           OriginalPath
+   end.
+
```

Comment:
`rebar_dir` already contains `format_source_file_name/2`; it requires 'opts' but supports `compiler_source_format` options from the compiler.

We should make a 1-arity wrapper that sends in empty opts and reuse it.